### ISSUE TITLE
 [UI] Footer Content Alignment Enhancement in KubeFirst Documentation

### ISSUE THREAD

**Issue by ATREAY** [kubefirst/kubefirst#1840]
### What is your feature idea?

The idea is to align all the content in the footer of KubeFirst documentation in a manner similar to the KubeFirst website.

Please find below an image depicting the footer that requires enhancement:
![footer](https://github.com/kubefirst/kubefirst/assets/66585295/ab6d60d4-821b-498d-8d34-ae3a03fbdb2b)

### Why is it needed?

To enhance user experience and maintain visual consistency, making the documentation more visually appealing.

**I am willing to work on this issue if it is considered a valid improvement.**

### Is this missing feature preventing you from using kubefirst?

- [ ] Yes

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fharper** [comment]
@ATREAY: thanks for reporting. I never thought about it, but you're right, the size of the cloud boxes should be the same, it will look better :)

**Comment by ATREAY** [comment]
Sure thing @fharper, should I go ahead and get started on it?

**Comment by fharper** [comment]
@ATREAY: yes, in that case, all cloud boxes should be the same size, not just the bottom 3.

**Comment by ATREAY** [comment]
Hey @fharper, I've got a quick question - do we keep the content inside the cloud box as it is, or should we adjust it to fit the box size?

**Comment by fharper** [comment]
We keep the content as it is, so adjust the box size to fit the content.

**Comment by ATREAY** [comment]
I just realized there's a different repo for the docs, but I raised the issue here. Is it acceptable if I connect this issue to that repo when I submit the PR?

**Comment by fharper** [comment]
I'll move the issue to the right repository, I didn't realize it was in the wrong one, no worries.
